userguide/sharing.rst

```diff
@@ -1379,7 +1382,7 @@ the administrator to control access to data, provide appropriate
 permissions to that data, and to determine who accesses and modifies
 stored data. A Windows domain controller is not needed for
 authenticated SMB shares, which means that additional licensing costs
-are not required. However, since there is no domain controller to
+are not required. However, when there is no domain controller to
```

Comment:
Maybe "because" would capture the meaning better.

Reply:
I believe that the whole heading "Configuring Authenticated Access Without a Domain Controller" is troublesome. What we're trying to show is "Configuring Authenticated Access with Local Users on the FreeNAS server". 

The current heading implies that if we have LDAP or NIS enabled, then we need to add local users as well (which we don't).